application/models/services/SurveyAggregateService/GeneralSettings.php

```
@@ -180,6 +180,16 @@ private function updateGeneralSettings(Survey $survey, array $input)
                 $survey
             );
 
+            $userSetting = \SettingsUser::getUserSetting('displayTimezone');
+            $tz = $userSetting->stg_value ?? \Yii::app()->getConfig('displayTimezone');
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$tz'. (undefined)

(UnusedLocalVariable)